models/stg_intercom__team_admin.sql

```diff
@@ -1,5 +1,5 @@
---To disable this model, set the using_team_admin variable within your dbt_project.yml file to False.
-{{ config(enabled=var('using_team', True)) }}
+--To disable this model, set the intercom__using_team_admin variable within your dbt_project.yml file to False.
```

Comment:
should this be `intercom__using_team_admin` or `intercom__using_team`?

Reply:
Thanks! must have been a mistake from doing the massive renaming.